src/dhcp6/fsm.c

```diff
@@ -2641,8 +2641,9 @@ ni_dhcp6_fsm_ia_addr_delete(ni_netdev_t *ifp, ni_dhcp6_device_t *dev, const ni_a
 		}
 	}
 
-	if (duplicate)
+	if (duplicate) {
```

Comment:
If I not mistaken, our code style is to avoid brackets on single statements

Reply:
yes.